### PR TITLE
Implement funcs to generate and check flip key word pairs

### PR DESCRIPTION
**Pull request by mbidenaio**

### CODE REVIEW

File: core/flip/word.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+)
+
+const (
+	a = 1103515245
+	c = 12345
+	m = 1 << 16
+)
+
+func GeneratePairs(seed []byte, k vrf.PrivateKey, n, pairCount int) (nums []int, proof []byte) {
+	hash, proof := k.Evaluate(seed)
```
[the remark this answers is not in the document]
Comment (mbidenaio) [reply core/flip/word.go]: Fixed

File: core/flip/word.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+	"math/big"
+)
+
+const (
+	a = 1103515245
+	c = 12345
+	m = 1 << 16
+)
+
+func GeneratePairs(seed []byte, k vrf.PrivateKey, n, pairCount int) (nums []int, proof []byte) {
```
[the remark this answers is not in the document]
Comment (mbidenaio) [reply core/flip/word.go]: Fixed

File: api/dna_api.go [hunk 3]
```diff
@@ -216,13 +221,23 @@ func (api *DnaApi) Identities() []Identity {
...
+func (api *DnaApi) Identity(addressHex string) Identity {
+	var address common.Address
+	if len(addressHex) > 0 {
+		address = common.HexToAddress(addressHex)
+	} else {
+		address = api.GetCoinbaseAddr()
+	}
+	var flipKeyWordPairs []int
+	if address == api.GetCoinbaseAddr() {
+		flipKeyWordPairs = api.ceremony.FlipKeyWordPairs()
```
Comment (ridenaio) [inline api/dna_api.go]: This ‘if’ is unnecessary, you can include this line in ‘else’ branch above

Comment (barsukovskii) [reply api/dna_api.go]: Fixed

**Pull request merged**